applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -531,7 +531,7 @@ def AssembleTestSuites():
 
 if __name__ == '__main__':
     KratosMultiphysics.Logger.PrintInfo("Unittests", "\nRunning cpp unit tests ...")
-    run_cpp_unit_tests.run()
+    # run_cpp_unit_tests.run()
```

Comment:
@philbucher @KlausBSautter I think this is a mistake?

Also, I think something else funny has happened to structural explicit - my previous explicit FETI coupling examples are now failing (even when I disable coupling) due to element inversion (which didn't happen ~2-4weeks ago)

Reply:
It is, I thought I had reverted this

In any case, the CI is not affected by this, the C++ tests are run differently there (in fact having it executed here too is a hack until we have a better solution, currently the C++ test of Structural and other apps are executed twice in the CI)

Hm not sure what exactly happened in the last 2-4 weeks, except the changes for #7381. But I am fairly certain this shouldn't affect anything like this.
In any case lets speak about it